src/meeting/teams/speakersObserver.ts

```
@@ -0,0 +1,522 @@
+                    }
+                    return isSpeaking
+                }
+
+                function checkElementAndPseudo(el: HTMLElement): boolean {
+                    const style = window.getComputedStyle(el)
+                    const beforeStyle = window.getComputedStyle(el, '::before')
+                    const borderStyle = window.getComputedStyle(el)
+
+                    // Old teams - EXACT SAME AS EXTENSION
+                    if (el.getAttribute('data-tid') === 'participant-speaker-ring') {
+                        return parseFloat(style.opacity) === 1
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)